consensus/types/src/signed_execution_payload_envelope.rs

```diff
@@ -48,6 +48,26 @@ pub struct SignedExecutionPayloadEnvelope<E: EthSpec> {
 }
 
 impl<E: EthSpec> SignedExecutionPayloadEnvelope<E> {
+    /// Create a new `SignedExecutionPayloadEnvelope` from an `ExecutionPayloadEnvelope` and `Signature`.
+    pub fn from_envelope(envelope: ExecutionPayloadEnvelope<E>, signature: Signature) -> Self {
```

Comment:
This function is also not used so I would leave it out.

Reply:
i think this helper is just used later in this VC block production [PR](https://github.com/shane-moore/lighthouse/pull/15) here:
https://github.com/shane-moore/lighthouse/blob/4eac78c70d1b1cb046180d21b6f81ffa5d617d9a/validator_client/lighthouse_validator_store/src/lib.rs#L760-L784

perhaps I just add the `from_envelope` to that PR instead?